PR: https://github.com/odoo/odoo/pull/

From: 781a03b2bc33e2fba96ab85ac38696c0691e7629
From: Yannick Tivisse
Date: 2017-09-01 11:03:18

Breaking data model changes score: 5, change matches:
-    _inherit = 'res.config.settings'
-    security_lead = fields.Float(related='company_id.security_lead', string="Security Lead Time")
-    group_route_so_lines = fields.Boolean("Order-Specific Routes",
-    module_sale_order_dates = fields.Boolean("Delivery Date")
-    group_display_incoterm = fields.Boolean("Incoterms", implied_group='sale_stock.group_display_incoterm')

Total Changes: 89

[IMP] res_config: Update file names, xmlids, class names according to guidelines

Now that we only have one model (res.config.settings). Uniformize everything according to the guidelines.

================================= pseudo patch: =================================

--- a/addons/sale_stock/models/__init__.py
+++ b/addons/sale_stock/models/__init__.py
@@ -6,6 +6,5 @@ from . import procurement
 from . import product
 from . import res_company
 from . import sale_order
-from . import sale_config_settings
-from . import stock_config_settings
+from . import res_config_settings
 from . import stock

--- a/addons/sale_stock/models/stock_config_settings.py
+++ b/addons/sale_stock/models/res_config_settings.py
@@ -4,10 +4,14 @@
 from odoo import api, fields, models
 
 
-class StockConfigSettings(models.TransientModel):
+class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
-    security_lead = fields.Float(related='company_id.security_lead')
+    security_lead = fields.Float(related='company_id.security_lead', string="Security Lead Time")
+    group_route_so_lines = fields.Boolean("Order-Specific Routes",
+        implied_group='sale_stock.group_route_so_lines')
+    module_sale_order_dates = fields.Boolean("Delivery Date")
+    group_display_incoterm = fields.Boolean("Incoterms", implied_group='sale_stock.group_display_incoterm')
     use_security_lead = fields.Boolean(
         string="Security Lead Time for Sales",
         oldname='default_new_security_lead',
@@ -23,12 +27,12 @@ class StockConfigSettings(models.TransientModel):
             self.security_lead = 0.0
 
     def get_values(self):
-        res = super(StockConfigSettings, self).get_values()
+        res = super(ResConfigSettings, self).get_values()
         res.update(
             use_security_lead=self.env['ir.config_parameter'].sudo().get_param('sale_stock.use_security_lead')
         )
         return res
 
     def set_values(self):
-        super(StockConfigSettings, self).set_values()
+        super(ResConfigSettings, self).set_values()
         self.env['ir.config_parameter'].sudo().set_param('sale_stock.use_security_lead', self.use_security_lead)

--- a/addons/sale_stock/models/sale_config_settings.py
+++ b/None
@@ -1,14 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import fields, models
-
-
-class SaleConfiguration(models.TransientModel):
-    _inherit = 'res.config.settings'
-
-    security_lead = fields.Float(related='company_id.security_lead', string="Security Lead Time")
-    group_route_so_lines = fields.Boolean("Order-Specific Routes",
-        implied_group='sale_stock.group_route_so_lines')
-    module_sale_order_dates = fields.Boolean("Delivery Date")
-    group_display_incoterm = fields.Boolean("Incoterms", implied_group='sale_stock.group_display_incoterm')
